Fitness/Views/Cells/ProRoutineCollectionViewCell.swift

```diff
@@ -0,0 +1,168 @@
+        descriptionTextView.isScrollEnabled = false
+        descriptionTextView.isSelectable = false
+        descriptionTextView.textContainerInset = UIEdgeInsets.zero
+        descriptionTextView.textContainer.lineFragmentPadding = 0
+        contentView.addSubview(descriptionTextView)
+
+        routineTypeImage.contentMode = .scaleAspectFit
+        routineTypeImage.clipsToBounds = true
+        contentView.addSubview(routineTypeImage)
+
+        routineTypeLabel.font = ._12MontserratRegular
+        routineTypeLabel.textColor = .secondaryBlack
+        routineTypeLabel.textAlignment = .left
+        contentView.addSubview(routineTypeLabel)
+
+        titleLabel.font = ._16MontserratMedium
+        titleLabel.textColor = .fitnessBlack
+        titleLabel.textAlignment = .left
+        contentView.addSubview(titleLabel)
+    }
+
+    private func setupConstraints() {
+        let addButtonSize = 24
+        let cellPadding = 16
+        let routineTypeImageSize = 15
+        let routineTypeLabelHeight = 15
+        let routineTypeLabelLeftPadding = 8
+        let routineTypeTopPadding = 3
+        let titleLabelHeight = 20
+
+        addButton.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(cellPadding)
+            make.trailing.equalToSuperview().inset(cellPadding)
+            make.height.width.equalTo(addButtonSize)
+        }
+
+        titleLabel.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(cellPadding)
+            make.leading.equalToSuperview().inset(cellPadding)
+            make.trailing.lessThanOrEqualTo(addButton.snp.leading).offset(-cellPadding)
+            make.height.equalTo(titleLabelHeight)
+        }
+
+        routineTypeImage.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(cellPadding)
+            make.top.equalTo(titleLabel.snp.bottom).offset(routineTypeTopPadding)
+            make.height.width.equalTo(routineTypeImageSize)
+        }
+
+        routineTypeLabel.snp.makeConstraints { make in
+            make.leading.equalTo(routineTypeImage.snp.trailing).offset(routineTypeLabelLeftPadding)
+            make.top.equalTo(titleLabel.snp.bottom).offset(routineTypeTopPadding)
+            make.trailing.lessThanOrEqualTo(addButton.snp.leading).offset(-cellPadding)
+            make.height.equalTo(routineTypeLabelHeight)
+        }
+
+        descriptionTextView.snp.makeConstraints { make in
+            make.top.equalTo(routineTypeLabel.snp.bottom).offset(cellPadding)
+            make.leading.trailing.bottom.equalToSuperview().inset(cellPadding)
+        }
+    }
+
+    // MARK: - LAYOUT
+    override open func layoutSubviews() {
+        super.layoutSubviews()
+
+        layer.cornerRadius = 4.0
+        layer.masksToBounds = false
+        layer.borderColor = UIColor.fitnessMutedGreen.cgColor
+        layer.borderWidth = 1
+
+        applySketchShadow(color: .fitnessMutedGreen, alpha: 1, x: 0, y: 11, blur: 14, spread: -10)
```

Comment:
does this have to be inside `layoutSubviews()`? can we move it to the `init`?

Reply:
apparently? I'm not sure why, but I just moved them to the `init` and the shadows were messed up